service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -81,21 +83,20 @@ public WorkflowQueryResult runSummary(String runId)
     }
   }
 
-  public Object getOutputs(String id) throws ApiException, AzureAccessTokenException {
+  public Object getOutputs(String id) throws ApiException {
```

Comment:
Should these methods also be passing in the bearerToken?

Reply:
Do we need to be passing the bearerToken for functions using the read client?

Reply:
I would assume so? Not sure if the listener will allow requests to the reader cromwell without a token. @cjllanwarne any thoughts?

Reply:
It depends whether the traffic from CBAS to the reader needs to leave the app and come back in via the listener or not. Which depends on whether the app is set up with the cromwell URL being the full path or a relative in-app path. I _believe_ it's all in-app but could be wrong. In that case, no tokens would need to be forwarded on

Reply:
Ok, so then we could get rid of the `getReadApiClient` signature with a bearer token and remove it from L59 above?